app/models/role.rb

```diff
@@ -0,0 +1,6 @@
+class Role < ApplicationRecord
+  belongs_to :user
+  enum access_level: {guest: 0, user: 1, admin: 2 }
```

Comment:
Sorry, but my OCD kicked in :)
```suggestion
  enum access_level: { guest: 0, user: 1, admin: 2 }
```
rubocop really helps in catching and correcting simple mistakes like that ;)